src/Menu/Menu.tsx

```diff
@@ -63,6 +64,8 @@ export const Menu = React.forwardRef<HTMLDivElement, MenuProps>(
         fullScreenOnMobile={fullScreenOnMobile}
         ref={ref}
         setStyle={setStyle}
+        withOverlay={withOverlay}
```

Comment:
```suggestion
```

Reply:
I need to remove that overlay if openOnHover is true.